tests/python/cli/example_function.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2023 CVAT.ai Corporation
```

Comment:
Let's try to add a real example based on https://github.com/ultralytics/ultralytics. As you suggested, we can create a python module with a detection function that is based on ultralytics and use it to explain how to write such functions. Also, we can use it for a demonstration to our users.
Without a way to run the functionality out of the box, it will demotivate 90% of regular users.

Reply:
Added.